### PR TITLE
Course overview

### PR DESCRIPTION
**Pull request by tushar13293**
## Summary

- Dummy data added in en.json

- Entries added in course overview page based on this data

## Issue

Can't add icons.

<!-- please don't

### CODE REVIEW

**Comment by tushar13293** [comment]
Hey @jankapunkt  could you look into the code if everything works fine or if you need me to make any changes? @ImkeMey Could you please check if everything is alright in terms of UI?

File: imports/ui/pages/myclasses/myClasses.js [hunk 1]
```diff
@@ -1 +1,77 @@
...
+  },
+  notStartedCourses () {
+    const notStartedArray = []
+    const notStartedObject = en.courses.notStartedCourses
+    for (const key in notStartedObject) {
+      notStartedArray.push({ key: key, value: notStartedObject[key] })
+    }
+    return notStartedArray
+  },
+  courseCategories () {
```
Comment (jankapunkt) [inline imports/ui/pages/myclasses/myClasses.js:58]: The course categories should be defined in a separate module that exports this object, something like `export const CourseCategories= { ... } `

File: imports/ui/pages/myclasses/myClasses.js [hunk 2]
```diff
@@ -1 +1,77 @@
...
+    return [
+      { text: 'Running Courses' },
+      { text: 'Completed Courses' },
+      { text: 'Courses not started yet' }
+    ]
+  },
+
+  noEntries () {
```
Comment (jankapunkt) [inline imports/ui/pages/myclasses/myClasses.js:66]: This should be defined as an i18n string in the Template

File: package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "@babel/runtime": "^7.7.6",
     "bcrypt": "^4.0.1",
     "meteor-node-stubs": "^1.0.0",
+    "sass": "^1.26.3",
```
Comment (jankapunkt) [inline package.json:21]: Please remove sass https://docs.npmjs.com/cli/uninstall

File: resources/i18n/en.json [hunk 4]
```diff
@@ -36,5 +36,26 @@
     "myClasses": "my-classes",
     "class": "class",
     "user": "user"
+  },
+
+  "courses": {
```
Comment (jankapunkt) [inline resources/i18n/en.json:41]: Please do not place your example data in the translation data, it's not the purpose of the translation definitions to contain example data. Please use a Mongo.Collection, where you include example documents on client startup instead.

**Comment by tushar13293** [comment]
Hey Jan i have made necessary changes you asked for. Could you please check again if everything works?

**Pull request merged**